### PR TITLE
Feature/strm 1389

### PR DESCRIPTION
**Pull request by bvdeenen**
Data subjects

### CODE REVIEW

File: pkg/entity/data_subject/cmd.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	longDoc := `Deletes 1 or more data subjects from the Data Subjects Service.
+
+Returns the number of deleted key links that were associated with these data subjects.
+`
+
+	outputFormatFlagAllowedValues := []string{common.OutputFormatPlain,
+		common.OutputFormatJson, common.OutputFormatJsonRaw}
+	outputFormatFlagAllowedValuesText := strings.Join(outputFormatFlagAllowedValues, ", ")
+	command := &cobra.Command{
+		Use:               "data-subjects <data-subject-id>...",
```
Comment (bobvandenhoogen) [inline pkg/entity/data_subject/cmd.go:50]: `<data-subject-id>` implies 1 value right, rather have something like `[id1,id2,..]`

File: pkg/entity/keylinks/cmd.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package keylinks
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func ListCmd() *cobra.Command {
+	longDoc := `Retrieve keylinks associated with data subjects`
+	command := &cobra.Command{
+		Use:               "data-subject-keylinks <data-subject-id>...",
```
Comment (bobvandenhoogen) [inline pkg/entity/keylinks/cmd.go:10]: idem as other comment

File: pkg/entity/data_subject/printers.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+func availablePrinters() map[string]util.Printer {
+	return util.MergePrinterMaps(
+		util.DefaultPrinters,
+		map[string]util.Printer{
+			common.OutputFormatPlain + common.ListCommandName:       listPlainPrinter{},
+			common.OutputFormatPlain + "0" + common.ListCommandName: listPlain0Printer{},
+			common.OutputFormatTable + common.ListCommandName:       listPlainPrinter{},
+			common.OutputFormatPlain + common.DeleteCommandName:     deletePlainPrinter{},
+		},
```
Comment (bobvandenhoogen) [inline pkg/entity/data_subject/printers.go:34]: No table printer, just plain?

File: pkg/entity/data_subject/cmd.go (hunk 1 above)
Comment (bvdeenen) [reply pkg/entity/data_subject/cmd.go:50]: I see the `...` as multiple. Let's see what Unix does.
```
man ls

LS(1)                            User Commands                           LS(1)

NAME
       ls - list directory contents

SYNOPSIS
       ls [OPTION]... [FILE]...
```

File: pkg/entity/data_subject/printers.go (hunk 3 above)
Comment (bvdeenen) [reply pkg/entity/data_subject/printers.go:34]: yep, the data is just a single column

**Pull request merged**